constants/achievementType.ts

```diff
@@ -0,0 +1,5 @@
+enum AchievementType{
+    NEW_ACHIEVEMENT = 'NEW_ACHIEVEMENT',
```

Comment:
We should be more specific with the type... I think this one would be something like AchievementType.BEAT_100_LEVELS or whatever